internal/core/api.rs

```diff
@@ -470,6 +512,18 @@ pub enum WindowEvent {
     },
     /// The pointer exited the window.
     PointerExited,
+    /// A key was pressed.
+    KeyPressed {
+        // FIXME: use SharedString instead of char (breaking change)
+        /// The unicode representation of the key pressed.
```

Comment:
should reference the `Key` enum here, to mention keys that do not have direct unicode representation.

Reply:
Good idea. I think we should.